pegasus/test/test_email_preference_helpers.rb

```diff
@@ -1,19 +1,19 @@
 require_relative '../helper_modules/dashboard'
 require_relative './test_helper'
 require_relative '../helpers/email_preference_helpers'
+require_relative 'sequel_test_case'
 require 'timecop'
 
-class EmailPreferenceHelperTest < Minitest::Test
+class EmailPreferenceHelperTest < SequelTestCase
```

Comment:
I'm not sure, I'd expect that to work. Could it have something to do with updating the Dashboard db rather than Pegasus? Do other existing test cases roll back correctly?

Reply:
`pegasus/test/test_form_helpers.rb` leaves behind rows in `pegasus_test.forms` when I run just that test (`rake test TEST=./test/test_form_helpers.rb`), but now when I run all Pegasus tests (`rake test`).

`pegasus/test/test_email_preference_helpers.rb` leaves behind rows in `dashboard_test.email_preferences` when I run all Pegasus tests AND when I run just that one test file.  I re-wrote the tests so that they still exercise the same rules without depending on the prior state of the database.